mongo-helm/templates/mongo.yml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mongodb-deployment
+  labels:
+    app: {{ .Values.appName}}
+spec:
+  replicas: 1

```

Comment:
We should be able to set replica count from values file. If not set only then a default value is used. Please don't hard code such things.

Reply:
Fixed.